### ISSUE TITLE
Proxychains

### ISSUE THREAD

**Issue by grizzpro** [robertdavidgraham/masscan#414]
Hi, does masscan work with Proxychains (non-tor)?

Thanks

**Comment by RoganDawes** [comment]
No, masscan is incompatible with ProxyChains in any form, due to its use of a custom user-mode networking stack. You can try nmap full TCP connect scan with proxychains, though.
